cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,103 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import PageObject from '../support/PageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+
+const page = new PageObject();
+const settingsPage = new SettingsPageObject();
+const profilePage = new ProfilePageObject();
+
 describe('Settings page', () => {
+  let updatedUserData;
+  let userData;
+
   before(() => {
+    cy.task('db:clear');
 
+    cy.task('generateUserData').then((newUserData) => {
+      updatedUserData = newUserData;
+    });
+
+    page.visit('/user/register');
```

Comment:
```suggestion
    page.visit('/user/register');
```
do you need to visit register page?  you use cy.register command in the before each block, that register user via api and do not need registration page